core/lib/basic_types/Cargo.toml

```diff
@@ -12,3 +12,5 @@ categories = ["cryptography"]
 [dependencies]
 web3 = { version= "0.19.0", default-features = false, features = ["http-rustls-tls", "test", "signing"] }
 serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.107"
```

Comment:
keep the version just 1.0, otherwise it will be impossible to simple increase the minor patch